CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Mapbox welcomes participation and contributions from everyone.
 
 #### Bug fixes and improvements
 - :warning: Moved `MapboxAudioGuidanceButton` from Drop-in UI to Voice module. [#5874](https://github.com/mapbox/mapbox-navigation-android/pull/5874)
+- Updated `Onboard` router doesn't support EV route requests yet and from now on the attempts will explicitly fail with an appropriate message. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
```

Comment:
```suggestion
- `Onboard` router doesn't support EV route requests yet and from now on the attempts will explicitly fail with an appropriate message. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
```

Reply:
any line of the changelog is added `Added`, `Moved`, `Fixed`, and so on. let's keep the same approach here.

Reply:
I can't fully agree, first of all the sentence should read well. The prefix guidelines are just that, guidelines. We can update separately to avoid waiting for a CI cycle.

Reply:
```suggestion
- Changed `Onboard` router to fail fast an deliver an appropriate message when EV route requests are made, which are not supported yet. [#5905](https://github.com/mapbox/mapbox-navigation-android/pull/5905)
```
This could work.